packages/logs/README.md

```diff
@@ -479,9 +479,18 @@ if (window.DD_LOGS) {
 
 After the Datadog browser logs SDK is initialized, it is possible to:
 
-- Set the entire context for all your loggers with the `setLoggerGlobalContext (context: Context)` API.
-- Add a context to all your loggers with `addLoggerGlobalContext (key: string, value: any)` API.
-- Get the entire global context with `getLoggerGlobalContext ()` API.
+- Set the entire context for all your loggers with the `setGlobalContext (context: Context)` API.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we shouldn't use `Context` here.
```suggestion
- Set the entire context for all your loggers with the `setGlobalContext (context: object)` API.
```

We should fix it [in the `contextManager` too](https://github.com/DataDog/browser-sdk/blob/main/packages/core/src/tools/contextManager.ts#L29). This is because of [a TS limitation](https://github.com/microsoft/TypeScript/issues/15300): the `Context` interface is incompatible with other interfaces who don't have a key index, see [this example](https://www.typescriptlang.org/play?ts=4.7.4#code/KYDwDg9gTgLgBASwHY2FAZgQwMbDgYQhVHgG8AoOOAbRAC44BnGKZAcwF0HDiQYA1TABsArsHIBfcqEiw4MAJ5g8PVH0Gi8AXiYt2cAD5wkIgLYAjNIbjmIEIcExJrqki6JqYAQShRMC6xEkABNgdGRgYOsTISFpcGh4ZFQMHBUPEh8-AJJgEMY4LP8AHld1YTEAPjhSKXJ0IOwYBCImYBgAcSEIc2EymAAKbAy+bhGYAEoauvJktCxcOABZBS6evvGaynlMGEwGZlYkNklyB3hTVe7eoX6GFbWb-rgdUh29hgAiZubPuBnGO1HhteINLsDbuMJkA).